.devcontainer/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:16-buster-slim
+# install base development tools
+RUN apt-get update && apt-get install -y git ruby ruby-dev build-essential rsync chromium parallel python3
+
+# make /bin/sh run /bin/bash
+RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+
+RUN npm install -g jest
```

Comment:
```suggestion
```

Reply:
Removing jest installation from the Dockerfile will result in the **first** start of VScode in a fresh checkout fail with an ugly error message.
The vscode Jest plugin requires jest to be installed "on load of the plugin".
(After running "yarn install" the first time, it would go away.)

You are sure, we want that ?

Reply:
![image](https://user-images.githubusercontent.com/127361/142725554-4c2e5f04-cab7-43bb-b620-38190ea260ce.png)

Reply:
The issue with `npm install -g jest` is that we will have jest installed globally and it may have a different version than the one we expect from the package.json. I'd rather have this error once than causing issues that are hard to debug later on.

Reply:
Yes, I can see that argument. Thought about it myself.
I did not find an other soultion. The configuration options for jest extension do not help.